lib/headway_analysis/server.ex

```diff
@@ -0,0 +1,75 @@
+defmodule HeadwayAnalysis.Server do
```

Comment:
Why not just start tracking vehicles present and doing the logging in the `Signs.Realtime` struct instead of setting up these new servers?

Reply:
I thought about that. Since this seems like an orthogonal task that's independent from what the main sign processes do, I thought it would be cleaner to put the code in its own processes.

Reply:
Gotcha. Makes sense to me